example/game/src/main/java/no/nordicsemi/android/ble/example/game/timer/TimerView.kt

```diff
@@ -0,0 +1,62 @@
+package no.nordicsemi.android.ble.example.game.timer
+
+import androidx.compose.animation.core.LinearEasing
+import androidx.compose.animation.core.animateFloatAsState
+import androidx.compose.animation.core.snap
+import androidx.compose.animation.core.tween
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.CircularProgressIndicator
+import androidx.compose.material3.Text
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+
+
+@Composable
+fun ShowTimer(
+    key: Any,
+    duration: Long,
+    modifier: Modifier = Modifier,
+) {
+    Row(
```

Comment:
Question to @sylwester-zielinski can it be improved? I remember I did it with Himali and it looks scary.

Reply:
This timer should start counting when a question changed (a key here) and stay hidden until the next question.